bibtexparser/bibdatabase.py

```diff
@@ -66,6 +67,23 @@ def __init__(self):
         #: List of fields that should not be updated when resolving crossrefs
         self._not_updated_by_crossref = ['_FROM_CROSSREF']
 
+    def __eq__(self, other: Any) -> bool:
+        """Equality == method."""
+        if (
+            isinstance(other, BibDatabase)
+            and other.entries == self.entries
+            and other.comments == self.comments
+            and other.strings == self.strings
+            and other.preambles == self.preambles
+        ):
+            return True
+        else:
+            return False
+
+    def __ne__(self, other: Any) -> bool:
```

Comment:
I do not think we need this, as it equals the default behavior: https://stackoverflow.com/questions/4352244/should-ne-be-implemented-as-the-negation-of-eq. I would propose to just use the tests to make sure the default behavior is indeed applied.

Reply:
Done, thanks for the head's up, I'll stop including it in my classes ;)